my-dsl.md

```diff
@@ -1,38 +1,58 @@
 # Language
-_What is the name of the language? Link the name to its webpage 
-(if appropriate)._
+The [Kivy Language (kv lang)](http://kivy.org/docs/api-kivy.lang.html).
 
 
 # Domain
-_Describe the language's domain in five words._
+User interface/interactions in Kivy applications.
 
 
 # Computational model
-_What is the underlying computational model of this language? To answer this 
-question, provide a high-level description (no more than 100 words) of the 
-computation that occurs when someone executes a program in this language._
+This language can't be executed alone - it requires (at the very least) a small
+wrapper of Python code.  When executed it is used to describe an application's
+appearance and interactability.  This is similar to HTML/CSS in web 
+development, or QML for Qt, or FXML for JavaFX.  Thus the file is parsed, then
+used to render the application.
 
 
 # DSL-ness
-_Fowler writes about a spectrum of languages, from general-purpose languages to 
-"purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
-and why?_ 
-
+This is not a purely domain-specific language.  It is essentially a modified 
+version of the Python language spec - in fact a large number of its statements
+are valid Python statements.  Thus it can do an awful lot beyond just laying
+out the application.  That being said, it would/should rarely be used as
+anything more than a DSL.  It is not a particularly friendly format to just
+write code in, and is designed to work within the constraints of a Kivy
+application.
 
 # Internal or external?
-_Is the language implemented as an internal or external DSL? 
-Justify your answer._
+External.  While it is heavily based on the Python language spec and can
+contain valid Python code, it is not in and of itself a valid Python program,
+and requires its own parser to be used.
+
+_Question: I'm not sure how to actually classify this DSL.  It seems like a bit
+of a blend, considering that substantial portions of it are or can be valid
+Python code._
```

Comment:
I agree that this is a grey area.

When we explored the definition of DSL in class, we proposed that the design intent and usage patterns of a PL can influence whether it is a DSL. Under that understanding I would feel comfortable calling this a DSL.

For reference, most people call LaTeX a DSL, even though it is Turing complete.